lively.project/helpers.js

```diff
@@ -8,7 +8,7 @@ export function projectAsset (fileName) {
   // Afterwards, we find out from within which project we were called.
   // This is the project of which we need to construct the asset path!
   const callStack = (new Error()).stack;
-  const callingProject = callStack.match(/\(.*\/local_projects\/(?<callingProject>[a-zA-Z_\-\d]+)\/.*\.js/m)?.groups.callingProject;
+  const callingProject = callStack.match(new RegExp('\\(.*\\/local_projects\\/(?<callingProject>[a-zA-Z_\\-\\d]+)\\/.*\\.js')).groups.callingProject;
```

Comment:
It is unrelated. This is because for some reason, rollup and/or google closure wreck complete havoc on this regular expression, making it completely unsuable within the bundles. Formatting the regex as a string fixes it. I am pretty sure that this is a bug in either google closure or rollup, but I was too lazy to dig down to the root cause.

Reply:
That can very well be. I remember that we had something similar occurring when developing qinoq. IIRC, you and @Paula-Kli spend some time then to discover that Google Closure wrecked some regex literals when storing worlds.